main.tf

```diff
@@ -1,3 +1,14 @@
+locals {
+  all_secrets = merge(
```

Comment:
Could you please run a `terraform fmt`? We don't have formatting in pre-commit for these one-off repos yet